src/mbgl/renderer/render_orchestrator.cpp

```diff
@@ -85,6 +98,9 @@ class RenderTreeImpl final : public RenderTree {
     std::vector<std::unique_ptr<RenderItem>> sourceRenderItems;
     std::reference_wrapper<LineAtlas> lineAtlas;
     std::reference_wrapper<PatternAtlas> patternAtlas;
+    std::vector<std::reference_wrapper<RenderLayer>> layersNeedPlacement;
+    std::shared_ptr<const Placement> placement;
```

Comment:
Orchestrator can still update placement, right? If render items would be moved to another thread in the future, this should be fixed to avoid data races.

Reply:
Not really, let me add one more commit to make it more explicit